static/js/ducks/filter.js

```diff
@@ -0,0 +1,104 @@
+import * as API from "../API";
+import store from "../store";
+
+export const FETCH_FILTER = "skyportal/FETCH_FILTER";
+export const FETCH_FILTER_OK = "skyportal/FETCH_FILTER_OK";
+export const FETCH_FILTER_ERROR = "skyportal/FETCH_FILTER_ERROR";
+export const FETCH_FILTER_FAIL = "skyportal/FETCH_FILTER_FAIL";
+
+export const ADD_GROUP_FILTER = "skyportal/ADD_GROUP_FILTER";
+export const ADD_GROUP_FILTER_OK = "skyportal/ADD_GROUP_FILTER_OK";
+
+export const DELETE_GROUP_FILTER = "skyportal/DELETE_GROUP_FILTER";
+export const DELETE_GROUP_FILTER_OK = "skyportal/DELETE_GROUP_FILTER_OK";
+
+export const FETCH_FILTER_V = "skyportal/FETCH_FILTER_V";
```

Comment:
Does V stand for version here? Perhaps make that more explicit for folks like me who don't know what that means

Reply:
Forgot to remove this from this duck -- this is relevant for F, so will be part of a PR on F, with proper naming.